### DISCUSSION TITLE
A way to jump from diff (patch) view to the corresponding file currently checked out?

### DISCUSSION THREAD

**Discussion by poige** [tpope/vim-fugitive#2056] [Q&A] [answered]
Currently tapping on the paths in diff view gives that version content (which is expected), but would be nice to have some quick way to just open the file as it is now from there as well.

BTW, if I press `.` then it opens command line pre-populated with dir path, which is somewhat close but not enough.

**Comment by tpope** [comment, marked as answer]
Jump to the historical file with `<CR>`, then use `:Gedit` to jump to the work tree version.

**Reply by poige** [reply]
mm, 2 hops with :cmdline use. Would you consider a shortcut?

**Reply by tpope** [reply]
No, key bindings are in short supply, and I'm not convinced this is an operation that would be helpful to most users.

If you just want to jump to the file, and not take advantage of the line number preservation that the 2 step solution provides, then it should be pretty straightforward to create your own map, built on `.`.
